packages/zoe/test/unitTests/contracts/test-operaConcertTicket.js

```diff
@@ -0,0 +1,409 @@
+                        );
+                        t.equal(
+                          typeof complete,
+                          'function',
+                          'complete should be a function',
+                        );
+
+                        afterRedeem();
+
+                        const currentAllocation = getCurrentAllocation();
+
+                        t.equal(
+                          currentAllocation.Ticket.extent.length,
+                          3,
+                          `the auditorium offerHandle should be associated with the 3 tickets`,
+                        );
+
+                        return {
+                          publicAPI,
+                          _operaPayout: payout,
+                          _complete: complete,
+                        };
+                      },
+                    )
+                );
+              },
+            );
+        });
+    },
+  );
+
+  const alicePartFinished = contractReadyP.then(({ publicAPI }) => {
+    const ticketIssuer = publicAPI.getTicketIssuer();
+    const ticketAmountMath = ticketIssuer.getAmountMath();
+
+    // === Alice part ===
+    // Alice starts with 100 moolas
+    const alicePurse = moolaIssuer.makeEmptyPurse();
+    alicePurse.deposit(moolaMint.mintPayment(moola(100)));
+
+    // Alice makes an invite
+    const aliceInvite = inviteIssuer.claim(publicAPI.makeBuyerInvite());
+    return inviteIssuer
+      .getAmountOf(aliceInvite)
+      .then(({ extent: [{ instanceHandle: instanceHandleOfAlice }] }) => {
+        const { terms: termsOfAlice } = zoe.getInstance(instanceHandleOfAlice);
+        // Alice checks terms
+        t.equal(termsOfAlice.show, 'Steven Universe, the Opera');
+        t.equal(termsOfAlice.start, 'Web, March 25th 2020 at 8pm');
+        t.equal(termsOfAlice.expectedAmountPerTicket.brand, moola(22).brand);
+        t.equal(termsOfAlice.expectedAmountPerTicket.extent, moola(22).extent);
+
+        const availableTickets = publicAPI.getAvailableTickets();
+        // and sees the currently available tickets
+        t.equal(
+          availableTickets.length,
+          3,
+          'Alice should see 3 available tickets',
+        );
+        t.ok(
+          availableTickets.find(ticket => ticket.number === 1),
+          `availableTickets contains ticket number 1`,
+        );
+        t.ok(
+          availableTickets.find(ticket => ticket.number === 2),
+          `availableTickets contains ticket number 2`,
+        );
+        t.ok(
+          availableTickets.find(ticket => ticket.number === 3),
+          `availableTickets contains ticket number 3`,
+        );
+
+        // find the extent corresponding to ticket #1
+        const ticket1Extent = availableTickets.find(
+          ticket => ticket.number === 1,
+        );
+        // make the corresponding amount
+        const ticket1Amount = ticketAmountMath.make(harden([ticket1Extent]));
```

Comment:
Should these be amounts to start? I'm not sure of the answer

Reply:
Either could work
I don't see a particular benefit to using one over the other in this context